cockatrice/src/settingscache.cpp

```diff
@@ -89,6 +91,12 @@ void SettingsCache::setCardInfoViewMode(const int _viewMode) {
     settings->setValue("cards/cardinfoviewmode", cardInfoViewMode);
 }
 
+void SettingsCache::setHighlightWords(const QString _highlightWords) {
+    // Words are seperated by a comma and you can not use spaces in words
+    highlightWords = _highlightWords.simplified().replace(" ", "").split(",");
```

Comment:
Why is the settingscache performing the conversion? It feels a little out of place. You should pass in the correct object type here.